subsys/bluetooth/controller/Kconfig

```
@@ -584,25 +584,42 @@ config BT_CTLR_SDC_CIS_SUBEVENT_LENGTH_US
 config BT_CTLR_CHANNEL_SOUNDING
 	bool "Channel Sounding support [EXPERIMENTAL]"
 	select EXPERIMENTAL
+	help
+	  Enable Channel Sounding support.
 
 config BT_CTLR_SDC_CS_COUNT
 	int "Number of concurrent connections supporting CS procedure [EXPERIMENTAL]"
 	default 1
+	range 1 BT_MAX_CONN
 	depends on BT_CTLR_CHANNEL_SOUNDING
 	help
-	  Set the number of concurrent connections that can support Channel Sounding procedure.
+	  Set the number of concurrent connections that can support Channel Sounding.
+	  Needs to be equal to or less than BT_MAX_CONN
 
 config BT_CTLR_SDC_CS_MAX_ANTENNA_PATHS
 	int "Max number of Channel Sounding antenna paths supported by the controller [EXPERIMENTAL]"
 	default 1
 	range 1 4
 	depends on BT_CTLR_CHANNEL_SOUNDING
+	help
+	  Set the max number of antenna paths the controller supports. This differs from BT_CTLR_SDC_CS_NUM_ANTENNAS
+	  since you can run a setup where local controller only has one antenna, but peer controller has two antennas.
+	  In this configuration, set BT_CTLR_SDC_CS_NUM_ANTENNAS to 1 and BT_CTLR_SDC_CS_MAX_ANTENNA_PATHS to 2
+	  (since local controller can send to both of the peer antennas, creating a total of 2 antenna paths)
+	  This option is purely a RAM-usage optimisation. If this option is set to 4, the controller has to reserve space for
+	  a total of 5 measurements per CS-step. However, if you set this to 1, the controller will only reserve space for
+	  2 measurements per CS-step.
 
 config BT_CTLR_SDC_CS_NUM_ANTENNAS
 	int "Number of antennas used in Channel Sounding procedure supported by the controller [EXPERIMENTAL]"
 	default 1
 	range 1 BT_CTLR_SDC_CS_MAX_ANTENNA_PATHS
 	depends on BT_CTLR_CHANNEL_SOUNDING
+	help
+	  Set the max number of antennas supported by local controller. This differs from BT_CTLR_SDC_CS_MAX_ANTENNA_PATHS
```

I felt this help text was confusingly written, what do you think of this suggestion?

```suggestion
Sets the max number of antennas the controller supports. The number of antennas used during channel sounding depends on the tone antenna configuration index selected in the procedure parameters. For instance, a A4_B1 configuration requires four antennas in the local device, while a A1_B4 configuration requires only one. Switching between antennas is done automatically using GPIOs whenever this kconfig is given a non-default value.
```